code/game/machinery/computer/ftl_weapons.dm

```diff
@@ -127,7 +127,7 @@
 		if(target)
 			data["target"] = target.name
 			var/list/ship_components_list = list()
-			data["ship_components"] = ship_components_list
+			data["components"] = ship_components_list
```

Comment:
good enough for now we might want to change the .ract at some point but I can put it on my to-do-list for further combat revamp updates.